src/components/cursor-controller.js

```diff
@@ -255,77 +249,71 @@ AFRAME.registerComponent("cursor-controller", {
   },
 
   _handleTouchStart: function(e) {
-    if (!this.isMobile || this.hasPointingDevice) return;
-
-    const touch = e.touches[0];
-    if (touch.clientY / window.innerHeight >= 0.8) return true;
-    this.mousePos.set(touch.clientX / window.innerWidth * 2 - 1, -(touch.clientY / window.innerHeight) * 2 + 1);
-    this._updateRaycasterIntersections();
+    if (!this.isMobile || this.hasPointingDevice || this.activeTouch) return;
```

Comment:
can you define this.activeTouch up in init() with the other variables?

Reply:
I can, but is writing `this.activeTouch = null` better than having `this.activeTouch` be `undefined` when I only care if it's truthy?

Reply:
It's mainly because thats how all the other variables are defined in this file and seems reasonable to maintain that pattern.